<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>662</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>662</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAN Bootloader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>317</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Node List</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="nodeListTableView">
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Operation Log</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTextEdit" name="logTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Device Index:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="deviceIndexComboBox">
              <property name="minimumSize">
               <size>
                <width>37</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Channel Index:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="channelIndexComboBox">
              <property name="minimumSize">
               <size>
                <width>37</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Baud Rate:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="baudRateComboBox">
              <property name="minimumSize">
               <size>
                <width>98</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>98</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>1000Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>800Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>666Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>600Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>400Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>300Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>250Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>150Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>90Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>80Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>60Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30Kbps</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20Kbps</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Start Addr(Hex):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startAddrLineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>98</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>08004000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Firmware File:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileLineEdit">
            <property name="maximumSize">
             <size>
              <width>236</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openFilePushButton">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Open File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updatePushButton">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="executePushButton">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Execute</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionContact_Us"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionContact_Us">
   <property name="text">
    <string>Contact Us</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
